Loop/Managers/DeviceDataManager.swift

```diff
@@ -1308,3 +1324,23 @@ extension DeviceDataManager: SupportInfoProvider {
     }
     
 }
+
+extension DeviceDataManager: PreferredGlucoseUnitPublisher {
+
+    func addPreferredGlucoseUnitObserver(_ observer: PreferredGlucoseUnitObserver, queue: DispatchQueue = .main) {
```

Comment:
No clients are requesting other queues; the queue dispatching of this should be removed.